### DISCUSSION TITLE
Styles to use Bootstrap 4

### DISCUSSION THREAD

**Discussion by Wellyson-Lopes** [josefarias/hotwire_combobox#175] [General]
Hello @josefarias, congratulations on your work, it really turned out great and I will use it in my project. However, I would like to know from you how I can configure the SCSS styles to use Bootstrap 4. Thank you and congratulations on your project.

**Comment by josefarias** [comment]
Hey @Wellyson-Lopes, thanks!

I'm not sure I completely understand. If you mean you want to use Bootstrap classes on the combobox — this is not documented yet but it's possible by customizing the appropriate elements. Here's a test for it in lieu of docs: https://github.com/josefarias/hotwire_combobox/blob/6fff0475ffc0c6cd6faf13b29eac11ded52b5647/test/dummy/app/views/comboboxes/custom_attrs.html.erb#L1-L20

If you mean you just want to use SCSS to style the combobox, I'd refer you to the Styling section of the docs in https://hotwirecombobox.com/. It should just be a matter of overriding the default classes with plain CSS (or SCSS).

Closing for now but let me know if this doesn't work for you, or if I misunderstood!
